src/Lykke.Service.PayInternal/Controllers/MerchantWalletsController.cs

```diff
@@ -73,19 +75,19 @@ public class MerchantWalletsController : Controller
             }
             catch (InvalidOperationException e)
             {
-                _log.WriteError(nameof(Create), request, e);
+                _log.Error(e, request);
```

Comment:
Alexey Novicikhin usually suggests to save in context only data wich is used for searching by fields in kibana.